llarp/config/config.cpp

```diff
@@ -852,6 +887,46 @@ namespace llarp
             "Permanently map a `.loki` address to an IP owned by the snapp. Example:",
             "mapaddr=whatever.loki:10.0.10.10 # maps `whatever.loki` to `10.0.10.10`.",
         });
+    // extra [network] options
+    // TODO: probably better to create an [exit] section and only allow it for routers
+    def.addOptionComments(
+        "network",
+        "exit",
+        {
+            "Whether or not we should act as an exit node. Beware that this increases demand",
+            "on the server and may pose liability concerns. Enable at your own risk.",
+        });
+
+    def.addOptionComments(
+        "network",
+        "auth",
+        {
+            "authenticate remote sessions against a whitelist or an external lmq server",
+            "true/false",
+        });
```

Comment:
This feels a little bit more complicated that necessary because we support *both* an LMQ auth server and whitelists.  Simpler is better here: we should support just whitelists for a "simple mode", and for anything else you set up an LMQ auth server (and if it wants to do whitelists plus other stuff, great, go ahead).

So this option, instead of true/false, can be "whitelist/lmq/none".  If whitelist then we use the `auth-whitelist` setting; if `lmq` we use lmq, if none we allow anyone.

Description could be:
"Authentication mode for remote sessions: can use a simple hard-coded whitelist, a separate lmq authentication server, or none to allow anyone."

I think this option either should not have a default at all (and so must be set explicitly whenever `network.exit` is set), or else should default to whitelist with an empty default whitelist (i.e. so it won't allow anything by default, even though exit mode is technically enabled).